cdc/kv/client.go

```diff
@@ -520,7 +520,12 @@ func (s *eventFeedSession) eventFeed(ctx context.Context, ts uint64) error {
 				s.errChSizeGauge.Dec()
 				allowed := s.checkRateLimit(errInfo.singleRegionInfo.verID.GetID())
 				if !allowed {
-					// rate limit triggers, add the error info to the rate limit queue
+					// rate limit triggers, add the error info to the rate limit queue.
+					log.Info("EventFeed retry rate limited",
+						zap.Uint64("regionID", errInfo.singleRegionInfo.verID.GetID()),
+						zap.Uint64("ts", errInfo.singleRegionInfo.ts),
+						zap.Int64("tableID", errInfo.singleRegionInfo.span.TableID()),
```

Comment:
Can log `span` info, besides table id has been stored in context, which can be used directly.